Softeq.NetKit.Chat.SignalR/Hubs/HubEvents.cs

```diff
@@ -5,7 +5,7 @@ namespace Softeq.NetKit.Chat.SignalR.Hubs
 {
     public static class HubEvents
     {
-        public const string MessageDeleted = "MessageDeleted";
+        public const string MessageDisabled = "MessageDisabled";
```

Comment:
I think that we should save endpoints and event names to avoid mobile side changes and hide implementation details

Reply:
Done